### ISSUE TITLE
[$50] Add new field "title" to the "Job" model

### ISSUE THREAD

**Issue by maxceem** [topcoder-platform/taas-apis#92]
We need to add a new field to the "Job" model to keep the job title:
- type: string, required, max length 64
- update swagger/postman
- create a migration script
   - for existent records populate it with the value from `description`
- update all related endpoints to support it
- update ES processor / es-mapping if needed

**Comment by maxceem** [comment]
Contest https://www.topcoder.com/challenges/30161319 has been created for this ticket.<br/><br/>```This is an automated message for maxceem via Topcoder X```

**Comment by maxceem** [comment]
@imcaizheng this is open for pick up now.

**Comment by maxceem** [comment]
Contest https://www.topcoder.com/challenges/30161319 has been updated - it has been assigned to aaron2017.<br/><br/>```This is an automated message for maxceem via Topcoder X```

**Comment by imcaizheng** [comment]
PR created https://github.com/topcoder-platform/taas-apis/pull/95

**Comment by imcaizheng** [comment]
PR for taas-es-processor https://github.com/topcoder-platform/taas-es-processor/pull/7

**Comment by maxceem** [comment]
@nkumar-topcoder also, during migration we have to update the mapping in ES similar to https://github.com/topcoder-platform/taas-apis/issues/22#issuecomment-737691144:
```
PUT /job/_mapping
{
  "properties": {
    "title": {
       "type": "text"
    }
  }
}
```

**Comment by maxceem** [comment]
Added a couple of small fixes 205f2646ea5164a9e650cd28f6cb1771380ee80d. Otherwise all works good.

**Comment by maxceem** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30161319<br/><br/>```This is an automated message for maxceem via Topcoder X```
